unlock-app/src/__tests__/middlewares/storageMiddleware.test.js

```diff
@@ -85,23 +86,33 @@ describe('Storage middleware', () => {
       expect(mockStorageService.storeTransaction).toHaveBeenCalledWith(
         transaction.hash,
         transaction.from,
-        transaction.to
+        transaction.to,
+        transaction.network
       )
       expect(next).toHaveBeenCalledTimes(1)
     })
   })
 
   describe('handling SET_ACCOUNT', () => {
-    it('should retrieve the transactions for that user', async () => {
-      expect.assertions(7)
+    it('should retrieve the transactions for that user and only dispatch the ones which match the network', async () => {
```

Comment:
Should we expect that this should filter out results based on the chain id? This may be a case of me misunderstanding the test description